src/features/firstLogin/achievements/pages/FirstTutorial/FirstTutorial.tsx

```diff
@@ -12,12 +12,15 @@ export function FirstTutorial() {
   function skip() {
     storage.saveObject('has_seen_tutorials', true)
   }
+
   return (
-    <GenericAchievement name="FirstTutorial" skip={skip}>
-      <FirstCard />
-      <SecondCard />
-      <ThirdCard />
-      <FourthCard />
-    </GenericAchievement>
+    <React.Fragment>
+      <GenericAchievement name="FirstTutorial" skip={skip}>
+        <FirstCard />
+        <SecondCard />
+        <ThirdCard />
+        <FourthCard />
+      </GenericAchievement>
+    </React.Fragment>
```

Comment:
Attention aux modifs de WIP ;)
```suggestion
      <GenericAchievement name="FirstTutorial" skip={skip}>
        <FirstCard />
        <SecondCard />
        <ThirdCard />
        <FourthCard />
      </GenericAchievement>
```